lua/gitblame/git.lua

```diff
@@ -75,20 +104,34 @@ end
 local function get_file_url(remote_url, branch, filepath, line1, line2)
     local repo_url = get_repo_url(remote_url)
     local isSrcHut = repo_url:find("git.sr.ht")
+    local isAzure = repo_url:find("dev.azure.com")
 
     local file_path = "/blob/" .. branch .. "/" .. filepath
     if isSrcHut then
         file_path = "/tree/" .. branch .. "/" .. filepath
     end
+    if isAzure then
+        -- Can't use branch here since the URL wouldn't work in cases it's a commit sha
+        file_path = "?path=%2F" .. filepath
+    end
 
     if line1 == nil then
         return repo_url .. file_path
     elseif line2 == nil or line1 == line2 then
+        if isAzure then
+            return repo_url .. file_path .. "&line=" .. line1 .. "&lineEnd=" .. line1 + 1 .. "&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents"
```

Comment:
Does the url always need to include the query params  `&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents`?

Reply:
Turns out that lineStyle and &_a were not needed, but lineStartColumn and lineEndColumn are unfortunately necessary, without it, the line highlights don't work :/

Reply:
Ah, I see. Azure can capture highlights starting and ending at different columns too. Makes sense :+1: